flink-connectors/flink-sql-connector-hive-3.1.3/src/main/resources/META-INF/NOTICE

```diff
@@ -25,25 +26,22 @@ the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 - javax.jdo:jdo-api:3.0.1
 - joda-time:joda-time:2.9.9
 - net.sf.opencsv:opencsv:2.3
-- org.apache.avro:avro-mapred:hadoop2:1.8.2
-- org.apache.avro:avro:1.8.2
```

Comment:
I believe we should retain these, since they are bundled into hive-exec also:

Extract from `hive-exec` build
```
[INFO] Including org.apache.avro:avro:jar:1.8.2 in the shaded jar.
[INFO] Including org.apache.avro:avro-mapred:jar:hadoop2:1.8.2 in the shaded jar.
```

Reply:
ok, the build does not allow the dependency to be defined twice:

```
2022-10-19T10:53:16.1186081Z 		 These issue are legally problematic and MUST be fixed: 
2022-10-19T10:53:16.1186514Z 			Dependency org.apache.avro:avro:1.8.2 is declared twice.
2022-10-19T10:53:16.1187176Z 			Dependency org.apache.avro:avro-mapred:hadoop2:1.8.2 is declared twice.
```